trade-finance-manager-api/src/v1/emails/amendments/manual-amendment-decision-email-variables.js

```diff
@@ -0,0 +1,123 @@
+const { AMENDMENT_UW_DECISION, AMENDMENT_TYPE } = require('../../../constants/deals');
+
+// email variables if amendment includes approved with conditions but not declined
+const approvedWithConditionsDecision = (amendmentVariables) => {
+  const { amendment, dealSnapshot, user } = amendmentVariables;
+  const { ukefFacilityId, ukefDecision } = amendment;
+  const { exporter, ukefDealId, bankInternalRefName } = dealSnapshot;
+  const { firstname, surname } = user;
+
+  return {
+    recipientName: `${firstname} ${surname}`,
+    ukefFacilityId,
+    exporterName: exporter.companyName,
+    bankReferenceNumber: bankInternalRefName,
+    ukefDealId,
+    conditions: ukefDecision.conditions,
+  };
+};
+
+// email variables if amendment is approved without conditions only
+const approvedWithoutConditionsDecision = (amendmentVariables) => {
```

Comment:
can we combine this function with the one from above? as far as I can see, the only difference is the `conditions` property

Reply:
done